app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java

```diff
@@ -138,7 +139,10 @@ public void createEvent() {
             .subscribe(createdEvent -> {
                 onSuccess.setValue("Event Created Successfully");
                 close.setValue(null);
-            }, Logger::logError));
+            }, throwable -> {
+                Logger.logError(throwable);
```

Comment:
Use timber

Reply:
Done

Reply:
Just for info, why timber here when the logError() function is also doing the same thing?